server/api/controllers/memberships/create-membership.js

```diff
@@ -0,0 +1,76 @@
+module.exports = {
+  friendlyName: "Create memberships",
+
+  description: "Create a member record",
+
+  inputs: {
+    officialName: {
+      type: "string",
+    },
+    userId: {
+      type: "string",
+    },
+    classDate: {
+      type: "json",
+    },
+    classAttendance: {
+      type: "boolean",
```

Comment:
how come this is type boolean if classDate is json(array of classDates). Shouldn't that mean classAttendance should be an array of boolean? or vice versa

Reply:
`classAttendance: {
"date": true
}`